test/client.js

```diff
@@ -2,6 +2,12 @@ const VError = require('verror');
 const net = require('net');
 const http2 = require('http2');
 
+const {
+  HTTP2_METHOD_POST,
+  // HTTP2_METHOD_GET,
+  // HTTP2_METHOD_DELETE
```

Comment:
I plan to use these in future tests, but commented them out to pass through the linter

Reply:
Could you add a short comment, why they are kept there